code/datums/mood_events/generic_negative_events.dm

```diff
@@ -239,6 +239,26 @@
 	mood_change = -25
 	timeout = 4 MINUTES
 
+/datum/mood_event/high_five_alone
+	description = "<span class='boldwarning'>I tried getting a high-five with no one around, how embarassing!</span>\n"
+	mood_change = -5
+	timeout = 1 MINUTES
+
+/datum/mood_event/high_five_full_hand
+	description = "<span class='boldwarning'>Oh God, I don't even know how to high-five correctly...</span>\n"
+	mood_change = -15
+	timeout = 2 MINUTES
+
+/datum/mood_event/left_hanging
+	description = "<span class='boldwarning'>But everyone loves high fives! Maybe people just... hate me?</span>\n"
+	mood_change = -10
+	timeout = 2 MINUTES
+
+/datum/mood_event/too_slow
+	description = "<span class='boldwarning'>NO! HOW COULD I BE.... TOO SLOW???</span>\n"
+	mood_change = -20 // the ULTIMATE prank carries a lot of weight
+	timeout = 2 MINUTES
```

Comment:
memes aside, i'd avoid powercreeping these values too much

as it stands getting too-slow'd is twice as bad as being actively held at gunpoint for your mood

Reply:
tbh that applies to real life too